extensions/azurecore/src/azureResource/interfaces.ts

```diff
@@ -40,3 +40,9 @@ export interface IAzureResourceNodeWithProviderId {
 export interface IAzureResourceService<T extends azureResource.AzureResource> {
 	getResources(subscriptions: azureResource.AzureResourceSubscription[], credential: msRest.ServiceClientCredentials, account: Account): Promise<T[]>;
 }
+
+export class ObjectExplorerActionsContext implements ObjectExplorerContext {
```

Comment:
This doesn't seem necessary - ObjectExplorerContext already has all these properties. You can just do 

```
if(node instanceof TreeNode) {
...
} else {
...
}
```

since there's only two options (that we tell it about anyways)